src/otx/core/exporter/native.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Class definition for native model exporter used in OTX."""
+
+from __future__ import annotations
+
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import onnx
+import openvino
+import torch
+
+from otx.core.exporter.base import OTXModelExporter
+from otx.core.types.export import OTXExportPrecisionType
+
+
+class OTXNativeModelExporter(OTXModelExporter):
+    """Exporter that uses native torch and OpenVINO conversion tools."""
+
+    def __init__(
```

Comment:
I would rather move this constructor to the base class. For now it seems strange that base class doesn't have signature of the constructor.

Reply:
Maybe it can be done by my PR. I already wrote common part of both Exporters into base class.

Reply:
`OTXModelExporter` serves rather as an interface, I didn't suppose it has any attributes.
Perhaps, constructor can be created for code sharing purposes